web/client/configs/localConfig.json

```diff
@@ -412,7 +412,7 @@
       { "name": "WidgetsTray" }
     ],
     "desktop": [
-      "Details",
+      "Details","DynamicLegend",
```

Comment:
In https://github.com/allyoucanmap/MapStore2/commit/a11b6736dd0cc5175977ec75a212d3c7d205d309 this has been excluded because the default localConfig.json should not contain the plugin. This plugin will be configured only via context in the default product application